python/README.md

```diff
@@ -0,0 +1,45 @@
+# Welcome to the CUDA Quantum Python API
+
+## Installing 
+Programmers may use pip to handle the installation of the CUDA Quantum Python API 
+and any of the dependencies needed to do so. This may be done either from source or
+through our binary wheel distribution. We currently support the following operating 
+systems: Linux.
+
+### Pip Wheels
+Under Construction! Not yet distributing wheels.
+Programmers may install CUDA Quantum via the command line:
+```
+pip install cuda-quantum --user
+```
+Note the `--user` flag is required for users on Linux, to ensure the package
+is installed in the local `site-packages` folder.
+
+### Pip install from source
+You may also install from source as follows:
+```
+git clone https://github.com/NVIDIA/cuda-quantum.git
+cd cuda-quantum
+pip install . --user
+```
+This will install any dependencies and build the necessary pieces of the CUDA Quantum 
+repository through cmake. It will then install the `cuda-quantum` python package to your
+pip package manager.
+
+### Build entirely from source
+If you would like to avoid the use of pip, or you want to build the entire C++ API with the
+Python API, follow the installation instructions here (TODO). You may then export your python
```

Comment:
@bettinaheim Can you remind me of the proper procedure for linking to internal documentation?

Reply:
@bettinaheim Also, how do we properly add words to filter out from spellcheck in `spellcheck_config.yml`? I'm having issues with "subcommand" here

Reply:
For documentation that is not part of the docs page but only part of this repository, you can simply give a relative path. Does that help, or do you need an example? What documentation do you want to link to?

As long as the word exist in a dictionary and/or is easy enough to find with a quick online search, you can add the word [here](https://github.com/NVIDIA/cuda-quantum/blob/main/.github/workflows/config/spelling_allowlist.txt).